### PR TITLE
Support include_v flag

### PR DESCRIPTION
**Pull request by nurse**

### CODE REVIEW

File: lib/td/client/api/table.rb [hunk 1]
```diff
@@ -81,9 +82,19 @@ def update_schema(db, table, schema_json)
...
+    update_table(db, table, {:expire_days=>expire_days})
+  end
+
+  # @param [String] db
+  # @param [String] table
+  # @option opts [Fixnum] :expire_days days to expire table
+  # @option opts [Boolean] :include_v (true) include v column on Hive
+  # @option opts [Boolean] :detect_schema (true) detect schema on import
+  # @return [true]
+  def update_table(db, table, params)
```
Comment (uu59) [inline lib/td/client/api/table.rb]: Doc comment called `opts` but the arg name is `params`. Maybe the same name is better.

File: lib/td/client.rb [hunk 2]
```diff
@@ -89,8 +89,8 @@ def database(db_name)
   end
 
   # @return [true]
-  def create_log_table(db_name, table_name)
-    @api.create_log_table(db_name, table_name)
+  def create_log_table(db_name, table_name, params)
+    @api.create_log_table(db_name, table_name, params)
```
Comment (uu59) [inline lib/td/client.rb]: These 2 lines change are necessary?
Looks no code is using added `params` argument.
And I'm worrying about this change breaks here: https://github.com/treasure-data/td-client-ruby/blob/50631e6f9e4fb8615ef186689a3a9b35e0367cd4/lib/td/client/model.rb#L98-L102

File: lib/td/client.rb [hunk 3]
```diff
@@ -111,6 +111,16 @@ def update_schema(db_name, table_name, schema)
     @api.update_schema(db_name, table_name, schema.to_json)
   end
 
+  # @param [String] db
+  # @param [String] table
+  # @option opts [Fixnum] :expire_days days to expire table
+  # @option opts [Boolean] :include_v (true) include v column on Hive
+  # @option opts [Boolean] :detect_schema (true) detect schema on import
+  # @return [true]
+  def update_table(db_name, table_name, params)
```
Comment (uu59) [inline lib/td/client.rb]: Doc comment called opts but the arg name is params. Maybe the same name is better.

File: lib/td/client.rb [hunk 4]
```diff
@@ -88,9 +88,14 @@ def database(db_name)
...
+  # @param [String] db
+  # @param [String] table
+  # @option params [Fixnum] :expire_days days to expire table
+  # @option params [Boolean] :include_v (true) include v column on Hive
+  # @option params [Boolean] :detect_schema (true) detect schema on import
   # @return [true]
-  def create_log_table(db_name, table_name)
-    @api.create_log_table(db_name, table_name)
+  def create_log_table(db_name, table_name, params={})
+    @api.create_log_table(db_name, table_name, params={})
```
Comment (uu59) [inline lib/td/client.rb]: `={}` isn't necessary here.

Comment (nurse) [reply lib/td/client.rb]: Indeed...
I added test for client.

**Pull request merged**